Edits to Migration Toolkit PR5658 (#5684)
EnterpriseDB/docs: product_docs/docs/migration_toolkit/55/09_mtk_errors.mdx

### Incorrect timestamps and Daylight Saving Time

> When migrating from SQL Server to PostgreSQL using the MSSQL JDBC driver, the migration may error without reporting any errors. However, if the source database contains rows with timestamp values that are within a range of time when Daylight Savings Time was in effect, those rows will be migrated to the target database with the wrong timestamps.

Reviewer: This was a bit confusing to me in both versions. How about:
```suggestion
When migrating from SQL Server to PostgreSQL using the MSSQL JDBC driver, the migration may not report any errors in spite of a timestamp migration failure. If the source database contains rows with timestamp values that are within a range of time when Daylight Savings Time was in effect, those rows will be migrated to the target database with the wrong timestamps.
```

Reviewer 2: ```suggestion
When migrating from SQL Server to PostgreSQL using the MSSQL JDBC driver, there can be an error in the migration that is not reported. Specifically, if the source database contains rows with timestamp values that are within a range of time when Daylight Savings Time was in effect, those rows will be migrated to the target database with the wrong timestamps.
```
